packages/cli/commands/lint.js

```diff
@@ -19,44 +20,47 @@ const i18nKey = 'cli.commands.lint';
 const { EXIT_CODES } = require('../lib/enums/exitCodes');
 
 exports.command = 'lint <path>';
-// Hiding since this command is still experimental
-exports.describe = null; //'Lint a file or folder for HubL syntax';
+exports.describe = 'Lint a file or folder for HubL errors and warnings';
 
 exports.handler = async options => {
-  const { path: lintPath } = options;
-
   await loadAndValidateOptions(options);
 
+  const { path: lintPath } = options;
   const accountId = getAccountId(options);
   const localPath = resolveLocalPath(lintPath);
+
+  trackCommandUsage('lint', null, accountId);
+
   const groupName = i18n(`${i18nKey}.groupName`, {
     path: localPath,
   });
-
-  trackCommandUsage('lint', null, accountId);
+  let errorCount = 0;
 
   logger.group(groupName);
-  let count = 0;
+
   try {
     await lint(accountId, localPath, result => {
-      count += printHublValidationResult(result);
+      errorCount = printHublValidationResult(result);
```

Comment:
`+=` removed because the `printHubLValidationResult` function returns a count of the errors, we don't need to do it here.

Reply:
Given the `exports.describe` - I'm assuming this was meant in the case of running the command on folders to add up the results count for each file.

Here I've added two test templates to `./src/templates` with the same deprecated module and run it on the folder before and after reverting the `+=`. Since whatever template it saw last had no issue it reports none found, even though it did find some.

![Screenshot 2023-09-25 at 2 19 45 PM](https://github.com/HubSpot/hubspot-cli/assets/25852903/cdbfc616-984e-40d1-a4e3-2be51789b7a5)

Related to the case of running it on folders, it would be a nice add to report which specific file the errors were found in, doesn't seem we do that now which makes them a quit a bit harder to pull apart